### PR TITLE
Feat: Select PDF from filesystem

### PR DESCRIPTION
**Pull request by dbjoEq**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -87,7 +86,8 @@
     "react-native-webview": "11.15.0",
     "react-query": "^3.23.2",
     "ts-jest": "^26.0.0",
-    "uuid": "^8.3.2"
+    "uuid": "^8.3.2",
+    "expo-document-picker": "~10.1.3"
   },
```
Comment (tormodAase) [inline package.json:91]: expo-image-picker can probably be removed from package.json. We don't use it anywhere else as far as I'm aware

**Review by tormodAase — CHANGES_REQUESTED** [review]
Looks good! The mad-expo-core import is a bit scary

File: package.json [hunk 2]
```diff
@@ -68,7 +67,7 @@
     "fbjs": "^3.0.0",
     "jest": "^26.6.3",
     "luxon": "^2.3.1",
-    "mad-expo-core": "git+https://github.com/equinor/mad-expo-core.git#dce42461c7a9b594f669ec38f961718a9b0fea4c",
+    "mad-expo-core": "git+https://github.com/equinor/mad-expo-core.git",
```
[the remark this answers is not in the document]
Comment (dbjoEq) [reply package.json]: Now pointing to newest main commit

File: package.json (hunk 1 above)
Comment (dbjoEq) [reply package.json:90]: Removed image picker!

**Review by tormodAase — DISMISSED** [review]

**Review by PtjeEquinor — APPROVED** [review]

**Pull request merged**